src/client/AppState.ts

```diff
@@ -86,6 +86,24 @@ export const PRISTINE_APPSTATE: AppState = {
       profileTypes: ['private', 'commercial'],
     }
   ),
+  PARKEREN_PRODUCTS: apiPristineResult(
+    {
+      data: [],
+    },
+    {
+      isActive: FeatureToggle.parkerenPatroonC,
+      profileTypes: ['private', 'commercial'],
+    }
+  ),
+  PARKEREN_PERMIT_REQUESTS: apiPristineResult(
+    {
+      data: [],
+    },
+    {
+      isActive: FeatureToggle.parkerenPatroonC,
+      profileTypes: ['private', 'commercial'],
+    }
+  ),
```

Comment:
Normaal gesproken voegen we geen aparte AppState secties toe voor specifieke stukjes data. Je zou dat dan toevoegen aan `PARKEREN` zoiets als:

```
PARKEREN: apiPristineResult({
  isKnown: true/false,
  vergunningen: [],
  vergunningsAanvragen: []
})
```

Oid.